src/test/java/org/jabref/model/entry/EntryLinkListTest.java

```diff
@@ -42,4 +67,102 @@ public void givenNullFieldValueAndDatabaseWhenParsingThenExpectLinksIsEmpty() {
         links = EntryLinkList.parse(null, database);
         assertTrue(links.isEmpty());
     }
+
+    @Test
+    public void givenTargetAndSourceWhenSourceCrossrefTargetThenSourceCrossrefsTarget() {
+        target = BibEntryBuild.ing().withId("target").withCiteKey("target").now();
```

Comment:
I don't see why it's necessary to have a BibEntry build class?
Why not just simply create two new BibEntrys? And you won't need the actual database file.

e.g.
```
database = new BibDatabase()
BibEntry source = new BibEntry()
source.setCiteKey("source")
source.setField(Crossref, "target")
database.insert(source)
BibEntry target = new BibEntry()
target.setCiteKey("target")
database.insert(target)

```

Reply:
Put that in the before method and it will be "freshly created" for every test

Reply:
`BibEntry` actually contains a `withField` method if you want to use a more fluent api,
https://github.com/JabRef/jabref/blob/23a0610ea7cba94172e5d9f0b4b6e0591003d667/src/main/java/org/jabref/model/entry/BibEntry.java#L734-L737
If you want, you can also add a new method `withCiteKey` along these lines.